src/smartcontracts/smartContractResults.local.net.spec.ts

```diff
@@ -58,8 +58,8 @@ describe("fetch transactions from local testnet", function () {
         await provider.sendTransaction(transactionDeploy);
         await provider.sendTransaction(transactionIncrement);
 
-        await watcher.awaitCompleted(transactionDeploy);
-        await watcher.awaitCompleted(transactionIncrement);
+        await watcher.awaitCompleted(transactionDeploy.getHash().hex());
```

Comment:
In a future PR, we should also adjust the integration tests to use the transaction factories for deploy, call etc.